menu-remove-element/README.md

```diff
@@ -0,0 +1,27 @@
+# menu-labelled-open
```

Comment:
Please update the title.

Reply:
Done. I have also expanded the last line.